src/EA.Weee.Web/Areas/Scheme/ViewModels/TransferSelectedDataModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace EA.Weee.Web.Areas.Scheme.ViewModels
+{
+    using System;
+    using System.Collections.Generic;
+
+    public class TransferSelectedDataModel
+    {
+        public IList<int> SelectedCategoryIds { get; set; }
+
+        public Guid SelectedSchemeId { get; set; }
+
+        public TransferSelectedDataModel(List<int> selectedCategory, Guid schemeId)
+        {
+            SelectedCategoryIds = selectedCategory;
```

Comment:
Condition that SchemeId is not empty and that selectedCategory is not an empty list. Use the new Cutting edge conditons.